cpp/ql/src/Security/CWE/CWE-190/ArithmeticUncontrolled.ql

```diff
@@ -59,10 +59,15 @@ predicate boundedBitwiseAnd(Expr e, Expr andExpr, Expr operand1, Expr operand2)
 }
 
 /**
- * Holds if `fc` is a part of the left operand of a binary operation that greatly reduces the range
- * of possible values.
+ * Holds if `e` is an operand of an operation that greatly reduces the range of possible values.
```

Comment:
In your new case `e` is the operation itself, not an operand (I don't think this is bad aside from not matching the QLDoc).

Reply:
Good catch! Fixed in bb6a2d3845a43b95c8dc23195857977e06087448.